orbit/cmd/orbit/orbit.go

```diff
@@ -622,6 +622,7 @@ func main() {
 		const (
 			renewEnrollmentProfileCommandFrequency = time.Hour
 			windowsMDMEnrollmentCommandFrequency   = time.Hour
+			windowsMDMEBitlockerCommandFrequency   = time.Hour
```

Comment:
Is the `E` after `MDM` a typo? If not I'm not sure what the `E` represents.

Reply:
Thanks, this was fixed [here](https://github.com/fleetdm/fleet/pull/14022/files#diff-78983f7f3415b23a44720d945f6286de2a4505298636b28571c26f6f4c570979R625)